src/components/Layouts/dashboardLayout.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function Layout(props) {
+  return (
+    <div className={props.class}>
+      <h1 align='center'> Welcome To Barefoot Nomad</h1>
+      {props.children}
+      <p align='center'> ---made by Elite Team---</p>
+    </div>
```

Comment:
This is not the one supposed to be a layout.
This is the one [supposed to be layout component](https://github.com/atlp-rwanda/elite-bn-fe/pull/36/files#diff-7902c3c268eb28dd7ec147d3c8bfd0d0c8b29c9e96d8f7ac193a571f078d8eb5)